app/serializers/region_serializer.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class RegionSerializer < ActiveModel::Serializer
+  attributes :key, :name, :created_at, :updated_at
+
+  has_many :locations
```

Comment:
Do we require the full location itself in the `included` section? As in to add something around the lines of `INCLUDED_ATTRIBUTES` and `include` in the controller? Or will we get that list in a different way maybe from a the single region?

Reply:
This will include only location id's as references - the front end already has full location details available via the `/reference/locations` endpoint and my understanding is that only the id's are needed to build region filtering against the `/allocations` endpoint. In future we can look at better supporting `includes` attribute to make this more flexible but that feels outside the scope of this PR/story.